src/application/write_to_sql.py

```diff
@@ -0,0 +1,20 @@
+from psycopmlutils.writers.sql_writer import write_df_to_sql
+
+import pandas as pd
+from pathlib import Path
+
+
+def load_split(split):
+    return pd.read_csv(SPLIT_PATH / (f"{split}_ids.csv"))
```

Comment:
Technically cannot save the IDs to Overtaci until the new DPIA is applied – sorry I didn't let you know :-)

Will just need to delete the ID-csvs from Overtaci for now.